python/whylogs/api/writer/whylabs.py

```diff
@@ -680,28 +700,26 @@ def _write_segmented_result_set(self, file: SegmentedResultSet, **kwargs: Any) -
 
         return and_status, "; ".join(messages)
 
-    def _tag_custom_output_metrics(self, view: Union[DatasetProfileView, SegmentedDatasetProfileView]):
-        if isinstance(view, DatasetProfileView):
-            column_names = view.get_columns().keys()
-            for column_name in column_names:
-                for perf_col in KNOWN_CUSTOM_OUTPUT_METRICS:
-                    if column_name.startswith(perf_col):
-                        data_type = KNOWN_CUSTOM_OUTPUT_METRICS[perf_col][0]
-                        discreteness = KNOWN_CUSTOM_OUTPUT_METRICS[perf_col][1]
-                        column_schema: ColumnSchema = ColumnSchema(
-                            classifier="output", data_type=data_type, discreteness=discreteness  # type: ignore
-                        )
-                        self._set_column_schema(column_name, column_schema=column_schema)
+    def _tag_custom_output_metrics(self, view: Union[DatasetProfileView, SegmentedDatasetProfileView, ResultSet]):
```

Comment:
```suggestion
    def _tag_custom_output_metrics(self, view: Union[DatasetProfileView, SegmentedDatasetProfileView]):
```

I don't think we need to handle `ResultSet` here